pos_session_close_by_tax/models/pos_order.py

```diff
@@ -0,0 +1,90 @@
+from odoo import models, _
+from odoo.tools import float_round
+
+
+class Order(models.Model):
+    _inherit = 'pos.order'
+
+    def _create_account_move_line(self, session=None, move=None):
+        res = super(Order, self)._create_account_move_line(session, move)
+        if move:
+            self._group_lines_by_tax(move)
+        return res
+
+    @staticmethod
+    def _check_grouping_applicability(lines):
+        list_of_date_maturity = lines.mapped('date_maturity')
+        if not all(elem == list_of_date_maturity[0] for elem in list_of_date_maturity):
+            return False
+        if len(lines.filtered(lambda x: x.tax_line_id).mapped('account_id')) > 1:
```

Comment:
```suggestion
        if len(lines.filtered('tax_line_id').mapped('account_id')) > 1:
```
If you wanna save some characters ;) not blocking of course
same goes for the line below